packages/lesswrong/lib/collections/automatedContentEvaluations/newSchema.ts

```diff
@@ -34,12 +34,44 @@ const schema = {
       typescriptType: "{ sentence: string; score: number; }[]",
```

Comment:
would be good to replace this with a reference to the `SentenceScore` type (which should exist due to codegen).

Reply:
(Happy to do this, but don't feel like I should have to tweak Habryka and Lucie's code to get an approval)